cps_data/cps_file_doc.md

```diff
@@ -0,0 +1,176 @@
+is then used to estimate the amount of premiums paid. The probability of a unit
+claiming this deduction is estimated based on the SOI Public Use File (PUF), and
+then the conditional amount claimed is then estimated as well.
+
+#### Other Imputations
+
+Capital gains/losses, taxable IRA distributions, adjusted IRA distributions,
+KEOGH plans, and student loan interest deductions are all imputed using a
+two-step approach based on the PUF. First, the probability of realizing a capital
+gain/loss, claiming a deduction, etc. is estimated. Then, conditional amounts
+for each field is imputed.
+
+### Final Preparations
+
+After adjusting for top-coding and imputations, we have a rough production file.
+The first step in readying the file for use with Tax-Calculator is simply
+renaming a number of variables so that they match the variable names in
+Tax-Calculator.
+
+Total wages, self-employment income, and farm income is found for each tax unit
+by summing the values for the head of the tax unit and their spouse.
+
+Interest income is split between taxable and nontaxable using the ratio between
+the two found in the PUF. Total dividends is separated into ordinary and
+qualified dividends in the same manor. Pensions and annuities included in AGI
+is derived from total pensions and annuities reported in the CPS also in
+proportion to the ratio in the PUF.
+
+We also apply the limitations to certain itemized deductions as they are found
+in current tax law. Total charitable contributions are capped at one half of AGI
+and, after this limit is applied, split between cash and non-cash contributions
+using the ratios found in the PUF.
+
+The student loan interest deduction is capped at $2,500, and IRA contributions
+are limited at either $6,500 or $5,500 depending on the age of the head of the
+tax unit.
+
+We then count the number of dependents under 5 and 13 or over 65 years old.
+Finally, the number of children qualified for the child tax credit, childcare
+credit, and earned income credit are counted for each tax unit.
+
+The last step in the final preparations is adjusting the distribution of interest
+income, ordinary and qualified dividends, and self-employment income using a
+method developed by OSPC. Using data from the Individual Statistical tables
+[made available by the IRS](https://www.irs.gov/uac/soi-tax-stats-individual-statistical-tables-by-size-of-adjusted-gross-income),
+we determine the distribution of each targeted variable as reported for the
+year 2014. The CPS records are then grouped into 19 bins based on total
+income.<sub>2</sub> Using the distribution found in the IRS data, we determine
+the ideal weighted sum in each income bin. This goal amount is then compared to
+the actual amount in each bin and a ratio is established for each bin. The
+records in each bin are then multiplied by their respective ratios so that the
+distribution more closely matches what is seen in the IRS data.
+
+### Tax-Calculator Integration
```

Comment:
Let's drop the tax-calculator integration section from this doc, which is about the file's prep.